### ISSUE TITLE
admin-react.js not found in v3.0.0 docker image

### ISSUE THREAD

**Issue by takeshi-yashiro** [jupyterhub/jupyterhub#4033]
### Bug description

The latest version of official docker image, `jupyterhub/jupyterhub:3.0.0` does not contain admin-react.js file, which results in blank hub admin page.

The lack of this file can be confirmed by entering the container and running the following command.

```
root@d82a49a1c06f:/srv/jupyterhub# cd /usr/local/share/jupyterhub/static/js
root@d82a49a1c06f:/usr/local/share/jupyterhub/static/js# ls -al
total 44
drwxr-xr-x 2 root root 4096 Sep  9 06:35 .
drwxr-xr-x 7 root root 4096 Sep  9 06:35 ..
-rw-r--r-- 1 root root 8013 Sep  9 06:35 admin.js
-rw-r--r-- 1 root root 3293 Sep  9 06:35 home.js
-rw-r--r-- 1 root root 4446 Sep  9 06:35 jhapi.js
-rw-r--r-- 1 root root  398 Sep  9 06:35 not_running.js
-rw-r--r-- 1 root root 1463 Sep  9 06:35 token.js
-rw-r--r-- 1 root root 4383 Sep  9 06:35 utils.js
```

I assume `yarn build` and `yarn place` need to be added in `Dockerfile`, as adding `admin-react.js` generated with these command like below solves this problem. 

```
COPY admin-react.js /usr/local/share/jupyterhub/static/js/admin-react.js
```

#### Expected behaviour

Access to  /hub/admin shows user management table, without any HTTP errors.

#### Actual behaviour

Access to  /hub/admin results in a blank page, with 404 error for /hub/static/js/admin-react.js.

### Your personal set up

 - OS: Docker Desktop v4.11.1 on macOS 12.5.1
 
 - Version(s): `jupyterhub/jupyterhub:latest` and `jupyterhub/jupyterhub:3.0.0` images
 

<details><summary>Full environment</summary>

```
# paste output of `pip freeze` or `conda list` here
```
</details>

<details><summary>Configuration</summary>

```python
# jupyterhub_config.py
```
</details>

<details><summary>Logs</summary>

```
# paste relevant logs here, if any
```
</details>

**Comment by benz0li** [comment]
> The latest version of official docker image, `jupyterhub/jupyterhub:3.0.0` does not contain admin-react.js file, which results in blank hub admin page.

@minrk I can confirm this issue.

**Comment by Paul2708** [comment]
Although the issue is fixed, it seems that the latest build of `jupyterhub/jupyterhub:3.0.0` does not contain the bug fix.
Is it planned to rebuild the image, or should we wait for the next release?

**Comment by consideRatio** [comment]
The image will be unchanged as we don't want to overwrite existing releases, so getting a 3.0.1 or 3.1.0 out is required for this.

**Comment by minrk** [comment]
I'll work on making a 3.1 release next week, I think

**Comment by benz0li** [comment]
I can confirm this is resolved with `jupyterhub/jupyterhub:3.1.0`.
